requirements.txt

```diff
@@ -1,8 +1,10 @@
-aiohttp==2.2.5
-async-timeout==1.2.1
-chardet==3.0.4
--e git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py
--e git+https://github.com/strinking/python-duckduckgo@async#egg=duckduckgo
-multidict==3.1.3
-websockets==3.4
-yarl==0.12.0
+PyYAML>=3.10
+aiohttp>=2.2.0
+async-timeout>=1.2.0
+chardet>=3.0.0
+multidict>=3.1.0
+ratelimit>=1.4.0
+websockets>=3.0
+yarl>=0.12.0
+git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice]
+-e git+https://github.com/strinking/python-duckduckgo@master#egg=duckduckgo-async
```

Comment:
the reason this is failing is
```
Cloning https://github.com/strinking/python-duckduckgo (to master) to /home/travis/virtualenv/python3.6.3/src/duckduckgo-async
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/home/travis/virtualenv/python3.6.3/src/duckduckgo-async/setup.py", line 2, in <module>
        from duckduckgo import __version__
      File "/home/travis/virtualenv/python3.6.3/src/duckduckgo-async/duckduckgo/__init__.py", line 11, in <module>
        from .query import query, get_zci
      File "/home/travis/virtualenv/python3.6.3/src/duckduckgo-async/duckduckgo/query.py", line 13, in <module>
        import aiohttp
    ModuleNotFoundError: No module named 'aiohttp'
```
because attempting to get the version via `from duckduckgo import __version__` runs this entire module
https://github.com/strinking/python-duckduckgo/blob/master/duckduckgo/__init__.py

if you want `duckduckgo.__version__` to be a thing, have a separate version.py file. in `__init__.py`, you can `from version import __version__` and from `setup.py` you can `from duckduckgo.version import __version__`

or you can just not make version a part of the lib because what is even the point

if you want, I can file this as a bug against py-ddg

Reply:
Ohh I see. Yes, please file it as an issue. I'd really like the mean red travis notifications to go away.